quipucords/scanner/network/normalizer.py

```diff
@@ -0,0 +1,78 @@
+"""normalizer for network scans."""
+
+import ipaddress
+
+from constants import DataSources
+from fingerprinter import formatters
+from scanner.normalizer import BaseNormalizer, FactMapper, NormalizedResult, str_or_none
+
+
+def infrastructure_type_normalizer(
+    *, virt_what_type=None, virt_type=None, subman_virt_is_guest=None
```

Comment:
Nice use of `*`. I wonder if there are more places we should use this to force better readability.